Content.Server/Damage/Components/DamageOnToolInteractComponent.cs

```diff
@@ -21,6 +21,12 @@ public class DamageOnToolInteractComponent : Component, IInteractUsing
         [DataField("tools")]
         private List<ToolQuality> _tools = new();
 
+        [DataField("weldingDamageType",required: true)]
+        private readonly DamageTypePrototype _weldingDamageType = default!;
+
+        [DataField("defaultDamageType",required: true)]
+        private readonly DamageTypePrototype _defaultDamageType = default!;
```

Comment:
Same problem of trying to resolve prototypes directly without prototype references.